README.md

```diff
@@ -37,6 +39,40 @@ You can use this [shell script][bootstrap-linux] to turn your laptop into an awe
 
 TODO
 
+## Run in Docker
+
+If you don't like to bootstrap your machine you can also run a [Docker][docker] container for development by invoking a single command only. The container will be pulled from our registry. You need to have [Docker Engine][docker-engine] installed first.
+
+~~~ bash
+# [pull and] run container with default rake task (eq. build `_site` and serve via jekyll)
+$ docker run --rm --volume=$(pwd):/usr/src/epages-docs -p 127.0.0.1:4000:4000 -it docker.epages.com/epages/docs-ruby
```

Comment:
Okay I will add the command to figure out the ip adress on a mac then below.. should be:

echo $(docker-machine ip `docker-machine active`)

will test this later. before changing the readme